sunpy/net/jsoc/jsoc.py

```diff
@@ -475,7 +475,8 @@ def get_request(self, requests, path=None, overwrite=False, progress=True,
         """
         c = drms.Client()
 
-        kwargs['max_splits'] = kwargs.get('max_splits', 2)
+        # Private communication from JSOC say we should not use more than one connection.
+        kwargs['max_splits'] = 1
```

Comment:
We should throw a INFO log here as well if the kwarg is passed.

Reply:
Fixed.